app/src/main/java/org/fossasia/openevent/app/events/EventsListAdapter.java

```diff
@@ -5,69 +5,41 @@
 import android.support.v4.app.FragmentManager;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.RecyclerView;
-import android.text.TextUtils;
 import android.view.LayoutInflater;
-import android.view.View;
 import android.view.ViewGroup;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.squareup.picasso.Picasso;
 
 import org.fossasia.openevent.app.R;
 import org.fossasia.openevent.app.data.models.Event;
+import org.fossasia.openevent.app.databinding.EventLayoutBinding;
```

Comment:
this is giving me error `Error:(13, 46) error: package org.fossasia.openevent.app.databinding does not exist`. Any hint?
@iamareebjamal